.github/workflows/trigger-circle-ci-workflow.yml

```diff
@@ -41,13 +42,14 @@ jobs:
             -H "Content-Type: application/json" \
             -H "Circle-Token: $CIRCLE_CI_TOKEN" \
             -d '{
-                  "branch": "${{ needs.get-branch.outputs.branch }}",
+                  "branch": "'"$BRANCH"'",
```

Comment:
This isn't remapped in the above `env` section. Did you miss that or is that on purpose?

Reply:
you're right. just saw the changes in the first job's env section xD